README.md

```diff
@@ -47,6 +47,29 @@ trigger.
 Notice the `Changes` tab at the job information, per each review an entry will
 be created.
 
+#### Remote Trigger
+
+Remote trigger is possible using webhook, URL is
+`https://jenkins/prefix/gerrit-webhook/`.
+
+Content:
+
+```json
+{
+  "project": {
+    "name": "project1"
+  }
+}
+```
+
+Example:
+
+```sh
+$ curl -X POST -d '{"project":{"name":"project1"}}' 'https://jenkins/prefix/gerrit-webhook/'
```

Comment:
This is what we consume *at the moment*. However, Gerrit webhooks provide a lot more information and we are going to leverage that information to optimise the associated Jenkins operation.

It would be best to include as well the references to the Gerrit webhook plugin and how to configure it.

Reply:
I had to reverse engineer the code to understand what it does, can you please merge this minimal hunk and then add more documentation? It is important to have documentation as people cannot guess what you mean when you write a code, even if it is great.